docs/openshift/kctlr-openshift-routes.rst

```diff
@@ -52,84 +50,107 @@ When you use the |kctlr| as a `Router`_, you can
 
    =======  ===================================================================
 
-.. _set up kctlr routes:
 
-Deploy the BIG-IP Controller
-----------------------------
+.. _route existing virtual:
 
-.. include:: /_static/reuse/kctlr-openshift-deployment-note.rst
+Attach Routes to Existing BIG-IP Virtual Servers
+------------------------------------------------
 
-#. Create a Kubernetes Deployment using valid YAML or JSON. Define the |kctlr| `Route configuration parameters`_ as appropriate to suit your needs.
+.. include:: /_static/reuse/k8s-version-added-1_5.rst
 
-   .. literalinclude:: /openshift/config_examples/f5-k8s-bigip-ctlr_openshift_routes.yaml
-      :linenos:
+If you need to use BIG-IP system functionality that isn't natively supported by the |kctlr|, you can attach a Route to an existing BIG-IP virtual server.
+Take the steps below **before** you deploy the |kctlr|.
 
-   .. warning::
+**If you want the |kctlr| to create a new virtual server for your Route,** :ref:`skip to the Basic Deployment section <kctlr routes basic>`.
 
-      Use caution when setting the :code:`--route-vserver-addr` and :ref:`specifying a BIG-IP SNAT pool <kctlr-openshift snat deploy>`.
+#. Create a virtual server in a BIG-IP partition that isn't already managed by a |kctlr| instance.
 
-      If you choose to set both options, make sure the IP address defined for the virtual server falls within the range of the selected SNAT pool.
+#. Customize the virtual server as needed. Be sure the settings applied don't conflict with those you want the Controller to apply for the Route.
 
-#. :ref:`upload openshift deployment`.
+#. In a TMOS shell, run the commands shown below to set the :code:`cccl-whitelist` metadata field. This field tells the Controller it should merge its configuration into the existing virtual instead of overwriting it.
 
-.. _route existing virtual:
+   - Make sure you're in the correct partition (for example, :code:`user@(BIG-IP)(cfg-sync Standalone)(Active)(/myPartition)(tmos)`).
+   - Replace "myVirtual" with the name of the virtual server on your BIG-IP device.
 
-Attach a Route to an Existing Virtual Server
-````````````````````````````````````````````
+   .. parsed-literal::
 
-.. include:: /_static/reuse/k8s-version-added-1_5.rst
+      modify ltm virtual **myVirtual** metadata add { cccl-whitelist { value 1 }}
```

Comment:
Would the next (last) step after this be to start the controller and specify the virtual on the command line (using either route-http-vserver or route-https-vserver?

Reply:
@russokj - That step is in the next section. I reorg'ed the doc so it has a better, more logical flow.